### ISSUE TITLE
Fundamentals 3 - Page 2 - Vis Basics

### ISSUE THREAD

**Issue by bcyee1** [sigmacomputing/sigmaquickstarts#106]
## TL;DR
Having followed the Quick Starts from 1 & 2 the order of the steps requires me to "redo" work as there is no `Profit` column that has been created in the `Data` sheet/tab.

## Steps
![Screenshot 2023-07-18 at 9 10 38 AM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/f3b43b8d-9a10-444c-8e2f-8a7cfa82ab8b)
1. From `Fundamentals 2 - Page 2 - Table Basics`. In this step we're making something _a child_ of `Data` called `Tables`.

![Screenshot 2023-07-18 at 9 12 40 AM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/abba37f4-9e8a-4c4b-8a84-45dbbeb4e969)
2. In `Fundamentals 3 - Page 2 - Viz Basics`. We see that we're going _back_ to the `Data` tab established in Fundamentals 1, which is just source & untransformed data.

![Screenshot 2023-07-18 at 9 13 35 AM](https://github.com/sigmacomputing/sigmaquickstarts/assets/139162552/5f6abcdc-3f63-409b-9961-54143af4ce02)
3. Later on in `Fundamentals 3 - Page 2 - Viz Basics` we see that `Profit` is referenced, which is a calculated column from `Fundamentals 2 - Page 4 - Calculated Columns` which was done in the `Table` tab.

## Recommended Action
Below are some options to improve the flow of the Quick Starts.

1. Alter `Fundamentals 2` so that there is a clearer flow, or optimized flow, between the Fundamental Exercises.
  - In `Fundamentals 2.2` name the 'source' or untransformed data as "source/raw"
  - In `Fundamentals 2.2` move the "create `Table` tab" instructions to `Fundamentals 2.5` or otherwise _after_ `Fundamentals 2.4` where the calculated columns are.
  - In `Fundamentals 2.2` make a 'child' of the "source/raw" and name it "Calculated Values" or something similar.
  - The remainder of `Fundamentals 2.2-2.4` happens on the "Calculated Values" table.
  - Reminder, the beginning of `Fundamentals 2.5` should have instructions to make a child of "Calculated Values" table in a new `Table` tab.
  - in `Fundamentals 3.2` note that you need to grab "Calculated Values" from the `Data` tab in the previous exercises.
2. Alternatively, add a link to Fundamentals 3 that goes to Fundamentals 2 to help a person remake the derived columns so that they can follow along
3. Alternatively, find a column that is not `Profit` that exists in the base data set that would make sense for this exercise.

**Comment by pballai** [comment]
We writing this content to include a profit column and making sure the workflow "works" between each fundamental QuickStart.
